src/tweet.js

```diff
@@ -19,7 +19,7 @@ export default class Tweet {
   }
 
   updateTweets(tweets) {
-    console.log('Displaying updated Tweets', tweets);
+    if (this.logger) {this.logger.error('Displaying updated Tweets');}
```

Comment:
@jungeunyoon Why is this logged as an error?

Reply:
@stulees ah man thanks.